_build/go.mod

```diff
@@ -2,7 +2,9 @@
 
 module github.com/os-observability/redhat-opentelemetry-collector
 
-go 1.20
+go 1.21
```

Comment:
@frzifus this was auto-generated. I thought that the go 1.20 is enforced by the OCB 0.93.0 that is pinned in the makefile

Reply:
I assume thats done by your local go version. What version do you have installed?

Reply:
Downgrading local go to 1.20 fixed it